notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        cur_flops = 2 * macs_dict[x.net_size] * visits_per_row * (cur_rows - prv_rows)
</pre><p>Conceptually I see why a MAC should be 2 FLOPS (multiply, 1, accumulate, 1) -- but your <code>df_mean</code> table gives basically the same results for ptflops and thop_macs -- so has this conversion already been applied somewhere else?</p><p> </p><p>(Probably doesn't matter too much so long as the computation is off by 2 consistently between this and the KataGo adv notebook though!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Ah df_mean table is actually entirely in units of macs.

Reply:
<p>OK, makes sense</p>